keyboards/vortex/m0110/ansi/m0110_625/info.json

```diff
@@ -0,0 +1,111 @@
+{
+    "manufacturer": "Vortex",
+    "keyboard_name": "M0110-625",
```

Comment:
```suggestion
    "keyboard_name": "M0110-625 ANSI",
```

Reply:
@dunk2k we want to keep keyboard name as "M0110_625", don't want ANSI, thanks.

Reply:
It can be helpful to users to distinguish between boards by their USB device name.